### ISSUE TITLE
Automation Support for Dropdown

### ISSUE THREAD

**Issue by andrewjtait** [Sage/carbon#1291]
Investigate support for automation and Dropdowns. Currently we render the options for the dropdown, even when they are hidden (for automation to work correctly). We should spend some time to investigate alternatives.

**Comment by SeanArmstrong** [comment]
We also need to investigate the accessibility of the dropdown that we have. Can the options even be accessed by page readers. Will it help to also render the options or will given focus to the dropdown allow for it to read all the options

**Comment by SeanArmstrong** [comment]
The reason for originally hiding this information is that we didn't need the markup if there was no focus. Performance reasons. If the dropdown items had lots of input listeners.

**Comment by ianforster-sage** [comment]
Fixed in PR #1690
